manifests/r/rocksdanister/LivelyWallpaper/2.0.2.1/rocksdanister.LivelyWallpaper.installer.yaml

```
@@ -15,20 +15,20 @@ UpgradeBehavior: install
 Dependencies:
   PackageDependencies:
   - PackageIdentifier: Microsoft.VC++2015-2019Redist-x86
-  - PackageIdentifier: Microsoft.DotNet.Runtime.3_1
+  - PackageIdentifier: Microsoft.DotNet.Runtime.6-x86
```

```suggestion
  - PackageIdentifier: Microsoft.DotNet.Runtime.6
```
DotNet PackageIdentifier does not contain architecture.
Unfortunately, currently there seems no way to specify architecture for Dependencies.

---

Maybe its better to just remove it rather than adding wrong arch? installer is going to handle it anyway.

---

Yeah, for now you can remove it. When winget can handle architecture for dependencies, we can add it back.